tests/test_app.py

```diff
@@ -223,7 +223,7 @@ def test_argparse(mocker):
     mock_expand = mocker.patch("os.path.expanduser", return_value=return_value)
     args = parser.parse_args([])
 
-    # check that the default home is used when no args args supplied
+    # check that the default home is used when no args are supplied
```

Comment:
This is an atypical correction...

Reply:
I love that you're doing this enough to spot which corrections are atypical. :green_heart:

Reply:
Oh, an atypical correction means I have to do manual work (thinking and correcting) instead of just using a default command that drops a duplication. The main tool just identifies problematic text—correcting it is left to supplemental stuff.

But, yes, I do their enough to have a sense of common errors including typos and doublings as well as generally incorrect corrections (when version is misspelled, Google Sheets success Verizon).